pkg/pgclient/config.go

```diff
@@ -2,14 +2,14 @@ package pgclient
 
 import (
 	"context"
+	"errors"
 	"flag"
 	"fmt"
-	"runtime"
-	"strconv"
-
 	"github.com/jackc/pgx/v4"
 	"github.com/timescale/promscale/pkg/log"
 	"github.com/timescale/promscale/pkg/pgmodel"
+	"runtime"
```

Comment:
Its a minor thing but imports need to be grouped by standard and 3rd party packages, like it was before this change.